google/cloud/spanner_v1/database.py

```diff
@@ -745,6 +798,16 @@ def execute_pdml():
 
         return _retry_on_aborted(execute_pdml, DEFAULT_RETRY_BACKOFF)()
 
+    @property
+    def _next_nth_request(self):
+        if self._instance and self._instance._client:
+            return self._instance._client._next_nth_request
+        return 1
+
+    @property
+    def _nth_client_id(self):
+        return self._instance._client._nth_client_id
```

Comment:
Is there a specific reason that we don't need a check like this here?

```python
if self._instance and self._instance._client:
```

Reply:
More of because we plumbed all the callers to have an _instance and _client: in test and then unlikely that either will be None when correctly setup. I however can make that change, no biggie.